src/RpcServer/Signers.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2015-2022 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Plugins
+{
+    class Signers : IVerifiable
```

Comment:
If we're moving this around, can we also rename this something more descriptive?